code/modules/reagents/reagents/medicine.dm

```diff
@@ -912,7 +912,7 @@ We don't use this but we might find use for it. Porting it since it was updated
 /datum/reagent/medicine/ethylredoxrazine
 	name = "Ethylredoxrazine"
 	id = "ethylredoxrazine"
-	description = "A powerful oxidizer that reacts with ethanol and overstimulation."
+	description = "Purges alcohol from the stomach per unit ingested. Also treats confusion, dizziness, sleepiness, and stuttering to sober up drunkards. Does nothing if injected."
```

Comment:
```suggestion
	description = "Purges alcohol from the stomach per unit ingested. Also treats confusion, dizziness, sleepiness, and stuttering to sober up drunkards. Does nothing if not injected."
```

Reply:
The sentences before mention ingestion. As in stomach glugg glugg. So I think that "Does nothing if injected" is correct, although... code removes dizziness, drowsiness, stuttering and confusion all the time. But it only removes alcohol if ingested.

Reply:
Wait. It does nothing **if the alcohol** was injected? Is that what was meant in the wiki?

Reply:
This is just confusing, I'll remove it.